lte/gateway/c/core/oai/tasks/nas5g/src/M5GDeRegistrationRequestUEInit.cpp

```diff
@@ -27,60 +27,47 @@ DeRegistrationRequestUEInitMsg::~DeRegistrationRequestUEInitMsg(){};
 
 // Decode De Registration Request(UE) Message and its IEs
 int DeRegistrationRequestUEInitMsg::DecodeDeRegistrationRequestUEInitMsg(
-    DeRegistrationRequestUEInitMsg* de_reg_request, uint8_t* buffer,
-    uint32_t len) {
+    uint8_t* buffer, uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.